recipes/repository.rb

```diff
@@ -29,7 +28,7 @@
 
 case node['platform_family']
 when 'debian'
-  include_recipe 'apt'
+  apt_update 'update'
```

Comment:
Docs say you can do
```suggestion
  apt_update
```
which sounds simpler to me

Reply:
The rspec test needs to check for a name, and it doesn't pass if it's not explicitly named: The docs say 'update' is the default name but it's not true at least on the version of chef I've tried.